packages/subapp-web/test/data/mock-app/dist/stats-with-runtime.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
you need to use stats from production build.  get it from `dist/server` by running `clap build`